OpenRA.Mods.RA2/Traits/BaseSpawnerParent.cs

```diff
@@ -225,16 +225,12 @@ public virtual void SpawnIntoWorld(Actor self, Actor child, WPos centerPosition)
 
 		void SetSpawnedFacing(Actor spawned, Actor spawner, Exit exit)
 		{
-			var facingOffset = facing == null ? 0 : facing.Facing;
-
-			var exitFacing = exit != null ? exit.Info.Facing : 0;
+			var facingOffset = facing?.Facing.Facing ?? 0;
+			var exitFacing = exit?.Info.Facing?.Facing ?? 0;
```

Comment:
These don't look right. I would expect this to look something like
```csharp
void SetSpawnedFacing(Actor spawned, Actor spawner, Exit exit)
{
	var facingOffset = facing?.Facing ?? WAngle.Zero;
	var exitFacing = exit?.Info.Facing ?? WAngle.Zero;

	var spawnFacing = spawned.TraitOrDefault<IFacing>();
	if (spawnFacing != null)
		spawnFacing.Facing = facingOffset + exitFacing;
}
```

This commit should also be merged into "Convert facing to WAngle"

Reply:
Sure looks better 👍